.styleci.yml

```diff
@@ -0,0 +1,8 @@
+enabled:
+
+disabled:
+  - align_double_arrow
```

Comment:
so the way consistent with symfony would be to leave this setting as it currently is, correct?